typescript/sdk/src/core/EvmCoreModule.ts

```diff
@@ -116,7 +143,7 @@ export class EvmCoreModule extends HyperlaneModule<
    * @returns Transaction that need to be executed to update the ISM configuration.
    */
   async createDefaultIsmUpdateTxs(
-    actualConfig: CoreConfig,
+    actualConfig: DerivedCoreConfig,
```

Comment:
The main difference is to make fields that we know are defined once the contracts are deployed on-chain also required on the type instead of having to check each time for if the field is undefined or not because we were using the "old" config schema where almost all the fields are optional